test/extended/authorization/authorization.go

```diff
@@ -1241,3 +1242,93 @@ var _ = g.Describe("[sig-auth][Feature:OpenShiftAuthorization] authorization", f
 		})
 	})
 })
+var _ = g.Describe("[sig-auth][Feature:OpenShiftAuthorization] ImageRegistry access", func() {
+    defer g.GinkgoRecover()
+    oc := exutil.NewCLI("bootstrap-policy")
+
+    g.Context("", func() {
+        g.Describe("PublicImageAccessWithBasicAuthShouldSucceed", func() {
+            g.It("should succeed [apigroup:image.openshift.io]", func() {
+
+                g.By("Enable default route for the image registry")
+                err := oc.AsAdmin().WithoutNamespace().Run("patch").Args(
+                    "configs.imageregistry.operator.openshift.io/cluster",
+                    "--type=merge",
+                    "-patch", `{"spec": {"defaultRoute": true}}`,
+                ).Execute()
+                o.Expect(err).NotTo(o.HaveOccurred())
+
+                g.By("Wait for the image registry to become available")
+                available := isImageRegistryAvailable(oc)
```

Comment:
You probably want to run this check before attempting to enable the default route, i.e as the first thing you do in your tests, then skip the tests if it's not _present_, and perhaps adding a wait/retry if it's progressing.